### DISCUSSION TITLE
How to send only 10sec of audio from esp32

### DISCUSSION THREAD

**Discussion by LokeshShelva** [pschatzmann/arduino-audio-tools#412] [Q&A] [answered]
I am using i2s microphone and the web server, to stream audio. How do I make it so that when client connects, only 10 secs of audio is streamed?

**Comment by pschatzmann** [comment, marked as answer]
You would study the AudioServer class and then overwite the doLoop method with your extended implementation

⁣Get TypeApp for Android ​

On Oct 16, 2022, 14:24, at 14:24, Lokesh Shelva P S ***@***.***> wrote:
>I am using i2s microphone and the web server, to stream audio. How do I
>make it so that when client connects, only 10 secs of audio is
>streamed? 
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/pschatzmann/arduino-audio-tools/discussions/412
>You are receiving this because you are subscribed to this thread.
>
>Message ID:
***@***.***>

**Reply by LokeshShelva** [reply]
ok. I was looking into the audio server implementation and I was thinking to override the method. I didn't know if there was better way to do it. Thank you.
